bin/ycsb.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/sh
```

Comment:
Are we expressly trying to keep this compatible with arbitrary shells? Personally, I think bash is prevalent enough to rely on it.